packages/@react-spectrum/s2/src/SelectBoxGroup.tsx

```diff
@@ -0,0 +1,446 @@
+        [noIllustration]: ['min-content']
+      }
+    }
+  },
+  gridTemplateColumns: {
+    orientation: {
+      horizontal: ['min-content', 12, '1fr']
+    }
+  },
+  alignContent: {
+    orientation: {
+      vertical: 'center'
+    }
+  },
+  borderRadius: 'lg',
+  borderStyle: 'solid',
+  borderColor: {
+    default: 'transparent',
+    isSelected: 'gray-900',
+    isDisabled: 'transparent'
+  },
+  backgroundColor: {
+    default: 'layer-2',
+    isDisabled: 'layer-1'
+  },
+  color: {
+    isDisabled: 'disabled'
+  },
+  boxShadow: {
+    default: 'emphasized',
+    isHovered: 'elevated',
+    isSelected: 'elevated',
+    forcedColors: 'none',
+    isDisabled: 'emphasized'
+  },
+  borderWidth: 2,
+  transition: 'default',
+  cursor: {
+    default: 'pointer',
+    isDisabled: 'default'
+  }
+}, getAllowedOverrides());
+
+const illustrationContainer = style({
+  gridArea: 'illustration',
+  alignSelf: 'center',
+  justifySelf: 'center',
+  minSize: 48,
+  color: {
+    isDisabled: 'disabled'
+  },
+  opacity: {
+    isDisabled: 0.4
+  }
+});
+
+const descriptionText = style({
+  gridArea: 'description',
+  alignSelf: 'center',
+  display: {
+    default: 'block',
+    orientation: {
+      vertical: 'none'
+    }
+  },
+  overflow: 'hidden',
+  textAlign: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+const labelText = style({
+  gridArea: 'label',
+  alignSelf: 'center',
+  justifySelf: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  width: '100%',
+  overflow: 'hidden',
+  minWidth: 0,
+  textAlign: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  fontWeight: {
+    orientation: {
+      horizontal: 'bold'
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+const gridStyles = style({
+  display: 'grid',
+  outline: 'none',
+  gridAutoRows: '1fr',
+  gap: {
+    gutterWidth: {
+      default: 16,
+      compact: 8,
+      spacious: 24
+    }
+  }
+}, getAllowedOverrides());
+
+/**
+ * SelectBox is a single selectable item in a SelectBoxGroup.
+ */
+export function SelectBox(props: SelectBoxProps): ReactNode {
+  let {children, value, isDisabled: individualDisabled = false, UNSAFE_style} = props;
+  
+  let {
+    orientation = 'vertical',
+    isDisabled: groupDisabled = false,
+    showCheckbox = false
+  } = useContext(SelectBoxContext);
+
+  const size = 'M';
+  const isDisabled = individualDisabled || groupDisabled;
+
+  return (
+    <ListBoxItem
+      id={value}
+      isDisabled={isDisabled}
+      textValue={value}
+      className={renderProps => (props.UNSAFE_className || '') + selectBoxStyles({
+        size, 
+        orientation, 
+        ...renderProps
+      }, props.styles)}
+      style={UNSAFE_style}>
+      {(renderProps) => (
```

Comment:
```suggestion
      {({isSelected, isDisabled, isHovered}) => (
```

nothing wrong with what you have, just make things a little more readable since you won't have to so renderProps.isSelected, renderProps.isDisabled, etc.